src/lib/components/drs/formComponents/DrsObjectChildren.js

```diff
@@ -24,6 +24,7 @@ const DrsObjectChildren = props => {
                     </Typography>
                 </div>
             }
+            activeDrsObject={{id: props.id}}
```

Comment:
can you change this prop to `activeDrsObjectId` since it's not the full `activeDrsObject` being passed?